pulsar-broker/src/main/java/org/apache/pulsar/broker/service/schema/SchemaRegistryStats.java

```diff
@@ -0,0 +1,157 @@
+import io.prometheus.client.Summary;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+class SchemaRegistryStats implements AutoCloseable {
+    private static final String SCHEMA_ID = "schema";
+    private static final String STATUS = "status";
+    private static final String CLUSTER = "cluster";
+    private static final String STATUS_SUCCESS = "success";
+    private static final String STATUS_FAILED = "failed";
+    private static final double[] QUANTILES = {0.50, 0.75, 0.95, 0.99};
+    private static final AtomicBoolean CLOSED = new AtomicBoolean(false);
+
+    private final Counter delSchemaOps;
+    private final Counter getSchemaOps;
+    private final Counter putSchemaOps;
+
+    private final Counter schemaIncompatibleCount;
+    private final Counter schemaCompatibleCount;
+
+    private final Summary delSchemaLatency;
+    private final Summary getSchemaLatency;
+    private final Summary putSchemaLatency;
+    private final String clusterName;
+
+    private static volatile SchemaRegistryStats instance;
+
+    static synchronized SchemaRegistryStats getInstance(final int interval, final String cluster) {
+        if (null == instance) {
+            instance = new SchemaRegistryStats(interval, cluster);
+        }
+
+        return instance;
+    }
+
+    private SchemaRegistryStats(final int interval, final String cluster) {
+        this.clusterName = cluster;
+
+        this.delSchemaOps = Counter.build("pulsar_schema_del_ops_count", "-")
+                .labelNames(CLUSTER, SCHEMA_ID, STATUS)
+                .create()
+                .register();
+        this.getSchemaOps = Counter.build("pulsar_schema_get_ops_count", "-")
+                .labelNames(CLUSTER, SCHEMA_ID, STATUS)
+                .create()
+                .register();
+        this.putSchemaOps = Counter.build("pulsar_schema_put_ops_count", "-")
+                .labelNames(CLUSTER, SCHEMA_ID, STATUS)
+                .create()
+                .register();
+
+        this.schemaCompatibleCount = Counter.build("pulsar_schema_compatible_count", "-")
+                .labelNames(CLUSTER, SCHEMA_ID)
+                .create()
+                .register();
+        this.schemaIncompatibleCount = Counter.build("pulsar_schema_incompatible_count", "-")
+                .labelNames(CLUSTER, SCHEMA_ID)
+                .create()
+                .register();
+
+        this.delSchemaLatency = this.setQuantiles(
+                Summary.build("pulsar_schema_del_ops_latency", "-")
+                        .maxAgeSeconds(interval).labelNames(CLUSTER, SCHEMA_ID));
+        this.getSchemaLatency = this.setQuantiles(
+                Summary.build("pulsar_schema_get_ops_latency", "-")
+                        .maxAgeSeconds(interval).labelNames(CLUSTER, SCHEMA_ID));
+        this.putSchemaLatency = this.setQuantiles(
+                Summary.build("pulsar_schema_put_ops_latency", "-")
+                        .maxAgeSeconds(interval).labelNames(CLUSTER, SCHEMA_ID));
```

Comment:
If we use summary here, we don't need to add _count above? The summary should expose the _count and _sum.

Reply:
It‘s not same, the delSchemaOpsCount record both failed and succeed ops count, distinguish with label `status`. 
But the summary only record succeed ops latency

Reply:
Reimplemented here, when get/del/put ops failed then record failed ops count, if succeed only record latency.